deploy/deploy.yaml

```diff
@@ -421,50 +421,50 @@ rules:
 apiVersion: rbac.authorization.k8s.io/v1
 kind: RoleBinding
 metadata:
-  name: sandboxed-containers-operator-leader-election-rolebinding
-  namespace: sandboxed-containers-operator-system
+  name: openshift-sandboxed-containers-leader-election-rolebinding
```

Comment:
Just curious if it is usual to keep a very long name even when there is a namespace?

In other words, do we want the whole name to be `openshift-sandboxed-containers-operator/openshift-sandboxed-containers-leader-election-rolebinding`, or the simpler `openshift-sandboxed-containers-operator/leader-election-rolebinding`?

Checking with pods in the various operator namespaces, I see no consistent pattern, but I don't see often `openshift` in the rolebinding names. Specifically, if I do `oc get rolebindings --all-namespaces`, this one stands out.

Reply:
The result is operator-sdk prefixing all those resources by default. One can disable this prefixing but that has other disadvanteges. Since nobody should have to type these things manually and in the webconsole they typically show up without the namespace in front I left the prefixing in.  Does that make sense?